File: _client/MyTeacherClient/src/app/ResponseModels/Role/GetRolesWithoutAdmin/GetRolesWithoutAdminResponse.ts
import BaseResponse from "../../../Helpers/BaseResponse/BaseResponse"

export class GetRolesWithoutAdminResponse extends BaseResponse<GetRolesWithoutAdminResponseModel[]>{
    
}
export class GetRolesWithoutAdminResponseModel{
    constructor(){
        this.name = ""
        this.shortCode = ""
    }
    id?: number
    name: string
    shortCode: string
}

File: _src/CoreService/Core/CoreService.Application/Features/Queries/Role/GetRolesExceptAdmin/GetRolesWithoutAdminResponse.cs
using MyTeacher.Helper.Responses;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace CoreService.Application.Features.Queries.Role.GetRolesExceptAdmin
{
    public class GetRolesWithoutAdminResponse : BaseResponse<List<GetRolesExceptAdminResponseModel>>
    {
    }

    public struct GetRolesExceptAdminResponseModel
    {
        public GetRolesExceptAdminResponseModel()
        {
            this.Name = string.Empty;
            this.ShortCode = string.Empty;
        }

        public string Name { get; set; }
        public string ShortCode { get; set; }
    }
}